lighthouse-core/test/config/config-test.js

```diff
@@ -727,6 +741,38 @@ describe('Config', () => {
       assert.strictEqual(config.categories['lighthouse-plugin-simple'].title, 'Simple');
     });
 
+    it('should load plugins from the config and from passed-in flags', () => {
+      const baseConfigJson = {
+        audits: ['installable-manifest'],
+        categories: {
+          myManifest: {
+            auditRefs: [{id: 'installable-manifest', weight: 9000}],
+          },
+        },
+      };
+      const baseFlags = {configPath: configFixturePath};
+      const simplePlugin = 'lighthouse-plugin-simple';
```

Comment:
nit: could we go with `PluginName` here? just easier to remember down below it's a string

Reply:
good call